root/static/scripts/release-editor/fields.js

```diff
@@ -800,6 +804,15 @@ class Release extends MB_entity.Release {
             });
         });
 
+        this.earliestYear = ko.computed(function () {
+            let years = [];
+            _(self.events()).each(function (event) {
```

Comment:
Instead of lodash I'd prefer using the native `.forEach` array method here, or better `.map`:

```JavaScript
return Math.min(
  ...self.events().map(e => e.unwrapDate().year),
);
```